client/payment-methods/index.js

```diff
@@ -287,7 +287,7 @@ const PaymentMethods = () => {
 						>
 							{ interpolateComponents( {
 								mixedString: __(
-									'The new WooPayments checkout experience will become the default on October 11, 2023.' +
+									'You were rolled back from dUPE to legacy UPE. Stay tuned on next actions.' +
```

Comment:
The wording is currently being aligned with the Design Team. paJDYF-9xf-p2#comment-20974 I'll update the text once confirmed.